src/homework4/task7.py

```diff
@@ -0,0 +1,16 @@
+"""
+Оглянемся назад. Числа
+Даны два натуральных числа. Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+num_1 = 18
+num_2 = 30
+while num_1 and num_2:
+    if num_1 > num_2:
+        num_1 %= num_2
+    else:
+        num_2 %= num_1
+if not num_1:
```

Comment:
boilerplate code. can be done a bit more clear. (we can always keep bigger number in apropriate variable). Overall, it's first solution without cheats. nice